corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -510,9 +508,9 @@ FormplayerFrontend.on('navigateHome', function() {
     FormplayerFrontend.regions.breadcrumb.empty();
     if (currentUser.displayOptions.singleAppMode) {
         appId = FormplayerFrontend.request('getCurrentAppId');
-        FormplayerFrontend.navigate("/single_app/" + appId, { trigger: true });
+        FormplayerFrontend.trigger("app:singleApp", appId);
```

Comment:
Offlined with ben. I changed this because if you navigate directly,  then the logic to load the new list of apps (or the new app) in this case isn't called, and without that this feature change doesn't work. The handler for `"app:singleApp"` immediately calls this same navigate line above, so there's no problem with the URL; Ben and I also confirmed this by testing locally.